code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -4,9 +4,18 @@
 	damage = 8
 	damage_type = TOX
 	nodamage = 0
-	knockdown = 100
-	stutter = 5
-	drowsy = 50
+	stamina = 60
+	eyeblur = 10
+	knockdown = 10
+	slur = 5
+
+/obj/item/projectile/energy/bolt/on_hit(atom/target, blocked = FALSE)
+	. = ..()
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		var/obj/item/I = C.get_active_held_item()
+		if(I && C.dropItemToGround(I))
```

Comment:
Does this affect things like cursed objects/arm attachments and tool mods?

Reply:
no

Reply:
Beats me. I'll test it out later.